package.json

```diff
@@ -35,8 +34,6 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.11",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "prettier": "^2.8.8",
-    "@vitejs/plugin-react": "^3.1.0",
-    "vite": "^4.1.4"
```

Comment:
Just curious why we are removing these dependencies

Reply:
They are only relevant to the demo-app project. Not needed at mono-repo level.